### ISSUE TITLE
Shrine title on login screen/menu/appbar should be read out by screen reader

### ISSUE THREAD

**Issue by rami-a** [material-components/material-components-flutter-gallery#437]
## Acceptance Criteria
* The Shrine title is read by screen reader on:
  * Login screen
  * Desktop side menu
  * Mobile app bar
